<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QnlyMainWindow</class>
 <widget class="QMainWindow" name="QnlyMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="regionBaseComboBox">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>510</y>
      <width>85</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>---------</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Create new...</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_Region"/>
   <addaction name="actionRemove_Region"/>
   <addaction name="actionShow_Hide_Grid"/>
   <addaction name="actionAction_Export"/>
  </widget>
  <action name="actionAdd_Region">
   <property name="icon">
    <iconset resource="../../../common-resources/images.qrc">
     <normaloff>:/icon/add-region-icon</normaloff>:/icon/add-region-icon</iconset>
   </property>
   <property name="text">
    <string>Add Region</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add Region&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionRemove_Region">
   <property name="icon">
    <iconset resource="../../../common-resources/images.qrc">
     <normaloff>:/icon/remove-region-icon</normaloff>:/icon/remove-region-icon</iconset>
   </property>
   <property name="text">
    <string>Remove Region</string>
   </property>
   <property name="toolTip">
    <string>Remove Region</string>
   </property>
  </action>
  <action name="actionShow_Hide_Grid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../common-resources/images.qrc">
     <normaloff>:/icon/grid-icon</normaloff>:/icon/grid-icon</iconset>
   </property>
   <property name="text">
    <string>Show/Hide Grid</string>
   </property>
  </action>
  <action name="actionAction_Export">
   <property name="icon">
    <iconset resource="../../../common-resources/images.qrc">
     <normaloff>:/icon/export-icon</normaloff>:/icon/export-icon</iconset>
   </property>
   <property name="text">
    <string>action_Export</string>
   </property>
   <property name="toolTip">
    <string>Take a picture of the current layout</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../common-resources/images.qrc"/>
  <include location="../../../common-resources/images.qrc"/>
  <include location="../../../common-resources/images.qrc"/>
 </resources>
 <connections/>
</ui>
